### DISCUSSION TITLE
Blog post permission system

### DISCUSSION THREAD

**Discussion by BrettHuang2019** [supabase/supabase#7914] [Questions]
Hi beautiful and smart people, 

I am not very experienced, but I would like to ask if it's possible to achieve a permissio system with supabase. Example:
User A: usera@email.com
Creates a blog post. 
He can set the accessibility of this post as either: 
- public
- private
- viewed by: userb@email.com (read only)

Is it feasible by only supabse (and sveltekit) ?

Thank you!

**Comment by wcooperscott** [comment]
Check out [this YouTube video](https://www.youtube.com/watch?v=Ow_Uzedfohk) — it looks like it's exactly what you're asking for!
 It is a video about Supabase row level security but he uses a blog post system as the example.

The video teaches you how to publish or unpublish a blog post; that may be what you're looking for when you say "private" or "public".
If it is not, let me know and I can send a moment to help you think about a structure SQL design to fit your project requirements.
